sentences/de-CH/_common.yaml

```diff
@@ -49,7 +51,7 @@ lists:
       - in: "(f|fahrenheit)"
         out: "Fahrenheit"
 expansion_rules:
-  name: "[(d'|d|de|dr|der|die|ds|s)] {name}"
+  name: "([(d'|d|de|dr|der|die|ds|s)] {name}|d'{name})"
```

Comment:
You can define `name` as this: `"[(d'[' ']|d |de |dr |der |die |ds |s )]{name}"`

Reply:
I see. Is there any rule what should be used? While your version is shorter,  it seems more difficult to read (for humans) due to the additional [].

Reply:
Nope, there is no rule for this.